package.json

```
@@ -22,6 +22,9 @@
     "start": "npm config set quiet=true; eleventy --serve",
     "lerna": "lerna",
     "test": "node scripts/test.js",
+    "test:watch": "npm config set group=default && web-test-runner --node-resolve --playwright --watch --group=$npm_config_group",
+    "test:build:watch": "npm run build && npm run test:watch",
```

```suggestion
    "test:build:watch": "npm run build && npm run test:watch",
```

I'm wondering how reusable this is since most often we're testing a single component and this doesn't allow us to pass in components.  Can we make the test command a script that accepts inputs so we can pass in a build flag or read in the CI env variable?

---

Updated per our discussion to support one or many individual components.